### ISSUE TITLE
USER ISSUE: NullReferenceException in MoveAsManyItemsAsPossible

### ISSUE THREAD

**Issue by THplusplus** [StrangeLoopGames/EcoIssues#5513]
**Version:** 0.7.2.0 beta

**Steps to Reproduce:**
Tried to put 10 Lumber into Anvil when carrying 20 Lumber. Recipe was Sawmill, 10 Lumber were the only materials missing from the recipe.

**Expected behavior:**
Not crashed

**Actual behavior:**
Caught exception:

Caught exception invoking RPC PlayerInteract on Player!

NullReferenceException (Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.)

```
Stack:
   bei Eco.Gameplay.Items.Inventory.MoveAsManyItemsAsPossible(Inventory destination, Func`2 sourceStackPredicate, User user)
   bei Eco.Gameplay.Components.StorageComponent.OnActRight(InteractionContext context)
   bei Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   bei Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   bei Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   bei Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   bei Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   bei Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   bei Eco.Shared.Networking.NetPort.ProcessBuffer()
   bei Eco.Shared.Networking.NetPort.Update()
   bei Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()
```

**Comment by DavidWFarley** [comment]
Experiencing the same NPE :

[43:35] Disconnecting client [reason: Caught exception:

<size=60.00%>Caught exception invoking RPC PlayerInteract on Player!

Object reference not set to an instance of an object.


Stack:
   at Eco.Gameplay.Items.Inventory.MoveAsManyItemsAsPossible(Inventory destination, Func`2 sourceStackPredicate, User user)
   at Eco.Gameplay.Components.StorageComponent.OnActLeft(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   at Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   at Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   at Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   at Eco.Shared.Networking.NetPort.ProcessBuffer()
   at Eco.Shared.Networking.NetPort.Update()
   at Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()
</size>]...

**Comment by SeanAtr0n** [comment]
fixed in 7.2.1
